src/transaction.js

```diff
@@ -19,7 +19,7 @@ Transaction.SIGHASH_NONE = 0x02
 Transaction.SIGHASH_SINGLE = 0x03
 Transaction.SIGHASH_ANYONECANPAY = 0x80
 
-Transaction.fromBuffer = function (buffer, __disableAssert) {
+Transaction.fromBuffer = function (buffer, __disableExcess) {
```

Comment:
@weilu the default is strict,  it is only in [specific cases](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/src/block.js#L47-L53) that you would want to do this.

`__noStrict` or `__lenient` works for me.